packages/fxa-settings/src/components/App/index.tsx

```diff
@@ -199,14 +208,21 @@ const AuthAndAccountSetupRoutes = ({
   // The approach here may change or we'll want to update other components to receive
   // `serviceName` from this instead of calling integration.getServiceName() within pages
   const [serviceName, setServiceName] = useState<MozServices>();
+
   useEffect(() => {
+    // Don't initialize until integration is ready!
+    if (!integration) {
+      return;
+    }
+
     (async () => {
```

Comment:
Do we still need this to be async?

Reply:
It doesn't. Nice catch.

Reply:
Spotted another spot with aync `getServiceName` 🙈 
https://github.com/mozilla/fxa/blob/e4cb9723b5c8aeb138502220018a0c737cf58df4/packages/fxa-settings/src/pages/ResetPassword/index.tsx#L53

Reply:
Me too :+1: . I caught a couple more and have pushed again.